pkg/plugins/resources/cargopackage/source_test.go

```diff
@@ -58,6 +58,20 @@ func TestSource(t *testing.T) {
 			expectedResult: "0.1.0",
 			expectedError:  false,
 		},
+		{
+			name: "Passing case of retrieving none existing crate-test from the filesystem index",
```

Comment:
```suggestion
			name: "Passing case of retrieving non existing crate-test from the filesystem index",
```

Reply:
That's hot it was and it failed the spelling check

Reply:
I'll update in a jiffy